app/(afterLogin)/(employeeInformation)/employees/manage-employees/[id]/_components/job/addEmployeeJobInfrmation/index.tsx

```
@@ -49,7 +58,16 @@
       onSuccess: () => {
         handleClose();
-        onInfoSubmition && onInfoSubmition();
+        
+        // Call the callback to refresh job information data
+        if (onJobInfoUpdated) {
+          setTimeout(() => {
+            onJobInfoUpdated();
+          }, 500);
+        }
+      },
+      onError: (error: any) => {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)